### DISCUSSION TITLE
Scripting new selfcoded checks (not tasks)

### DISCUSSION THREAD

**Discussion by sightkick** [amidaware/tacticalrmm#1434] [General]
To get e.g. the idle time of the system (which I can´t find in the gui) I want to build a check which sends a string which is shown in the "more info" area of the check (like check disk space check).
But I can´t find the info where to put script and which variables to use to create such an output.

![Xnip2023-02-12_21-08-23](https://user-images.githubusercontent.com/57866026/218334600-ac1b0760-5b58-4faa-bec7-4123f1232255.jpg)

**Comment by wh1te909** [comment]
Just print the idle time to stdout in your script and then add it as a script check and you'll see the output when you click on "Last Output" in the more info column.

**Reply by sightkick** [reply]
Thanks for the info. I already got this working but it´s just a click too much for me :-) Isn´t it possible to write it direct into this line like you did?

**Reply by wh1te909** [reply]
not atm
